OJP/OJP_Places.xsd

```diff
@@ -364,7 +369,7 @@
 					<xs:documentation>Rough estimate of the travel duration from the specified reference place to this exchange point.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="WaitDuration" type="xs:duration" default="PT0M" minOccurs="0">
+			<xs:element name="WaitDuration" type="xs:duration" minOccurs="0" default="PT0M">
```

Comment:
I really dislike this merge frenzy without approval from several people.
This merge reverted https://github.com/VDVde/OJP/pull/286 again.
Why?